charts/artifact-hub/templates/hub_ingress.yaml

```diff
@@ -11,11 +11,13 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  {{- if .Values.hub.ingress.defaultBackend -}}
```

Comment:
Can we name it `defaultBackendEnabled`? Just in case we need to make the default backend configurable at some point.